File: customer-module/backend/src/main/java/com/pos/customer/entity/Order.java
package com.pos.customer.entity;

import java.time.LocalDate;
import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Order {
	private Integer orderId;
	private LocalDate date;
	private Double totalPrice;
	private Float discount;
	private String modeOfPayment;
	private String status;
	private String tracking;
	private Address address;
	private Customer customer;
	private List<OrderItem> orders;
}

File: admin-module/frontend/src/app/services/staff/order/order.ts
import { Address } from "../../address/address.service";
import { Customer } from "../customer/customer.service";

export class Order {
    public id?:number;
    public date?:Date;
    public totalPrice?:number;
    public discount?:number;
    public modeOfPayment?:string;
    public status?:string;
    public tracking?:string;
    public customer?:Customer;
    public address?:Address;
}
